statistics/handle/handle.go

```diff
@@ -618,7 +617,7 @@ func (h *Handle) Update(is infoschema.InfoSchema, opts ...cache.TableStatsOpt) e
 		tbl.TblInfoUpdateTS = tableInfo.UpdateTS
 		tables = append(tables, tbl)
 	}
-	h.updateStatsCache(oldCache.Update(tables, deletedTableIDs, lastVersion, opts...))
+	h.updateStatsCache(oldCache.Update(tables, deletedTableIDs, opts...))
```

Comment:
What about adding logic here? If maxTableVersion loaded here is not larger than oldCache.Version, we do not do update.

Reply:
I've tried this but found some bad cases. For example, in the beginning, we have 2 tables, and their stats versions are `t1:100` and `t2:200` respectively, then the cache version should be `200`.
And then we drop the table `t2` and then try to update the cache, since we'll find we cannot update the cache, since cache version is `100` which is less than `200`.

Reply:
If we do not update cache version when Delete, why does cache version become 100 from 200?